server.js

```diff
@@ -125,24 +125,24 @@ app.put('/api/v1/projects/:id', async (request, response) => {
 app.put('/api/v1/palettes/:id', async (request, response) => {
   const palette = request.body;
 
-  for (let requiredParameter of ['name', 'color1', 'color2', 'color3', 'color4', 'color5', 'project_id']) {
-      if (!palette[requiredParameter]) {
-        return response
-          .status(422)
-          .send({ error: `Expected format: { title: <String>, author: <String> }. You're missing a "${requiredParameter}" property.` });
-      }
-    }
+  if (!Object.keys(palette).length) {
+    return response
+      .status(422)
+      .send({ error: `Expected format: { key: <Value> }. You've not added any data to change.`});
```

Comment:
How does this return the missing value?

Reply:
It doesn't return any value, It throws error if body object is empty and don't have any key for updating.